src/Symfony/Component/Console/Exception/CommandNotDefinedException.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Exception;
+
+/**
+ * Represents an incorrect command name typed in the console.
+ *
+ * @author Jérôme Tamarelle <jerome@tamarelle.net>
+ */
+class CommandNotDefinedException extends \InvalidArgumentException implements ExceptionInterface
```

Comment:
I would have named it `CommandNotFoundException` to be more consistent with other exception names in other components.